### DISCUSSION TITLE
Question about image optimization not working properly

### DISCUSSION THREAD

**Discussion by ceruberu** [vercel/next.js#20149] [Help]
I really tried to find everywhere but I couldn't find the answer,
so I'm here to ask you.

I'm using imgix as image cdn provider

<img width="1082" alt="스크린샷 2020-12-14 오전 1 55 58" src="https://user-images.githubusercontent.com/21168032/102021054-b29ae980-3daf-11eb-826c-abc35685c583.png">

As you see, I have view-width at 425, but I receive w=828 from 

```
   <div className="relative h-52 flex justify-center">
        <Image
            className="opacity-80"
            alt="Reviews Mosaic"
            src="/public/reviewCollage.jpg"
            layout="fill"
            objectFit="cover"
        />
        <div className="flex justify-items-center items-center">
           <div className="text-white z-10 font-bold text-2xl bg-gimme p-4 rounded-md shadow-lg ">
                  600+ <FM id="reviewText"/>
            </div>
         </div>
   </div>
```

Is it because I use layout fill?
If so, how do I get just the image size I really need?

docs say defaults for deviceSizes are 
[640, 750, 828, 1080, 1200, 1920, 2048, 3840]
So why am I not getting like 640?

I really looked through everywhere please help me
This is destryoing my Largest Contentful Paint
Thanks in advance

**Comment by tomafro** [comment]
How big is the source image, and what markup is generated by your `<Image>` tag? It's likely the browser is requesting a bigger width than you expect because the pixel density of your screen is > 1. On my browser, typing `window.devicePixelDensity` into the developer console returns 2; there are two _physical pixels_ for every _css pixel_ in each dimension. In order for images to appear sharp they need twice the resolution in both directions.
